haxepunk/math/MathUtil.hx

```diff
@@ -95,8 +95,7 @@ class MathUtil
 			return;
 		}
 		point.normalize(distance);
-		object.x += point.x;
-		object.y += point.y;
+		object += point;
```

Comment:
This is no longer modifying `object` in place.

Reply:
Oh yeah, true. Good thing is that I can just override the = operator for assignment !

Reply:
Well turns that a bug in Haxe doesn't allow me to do that, so I'll revert that as well.